server/pulp/server/managers/consumer/agent.py

```diff
@@ -416,6 +470,7 @@ def __init__(self, units):
         :param units: A list of content units.
         :type units: list
         """
+        super(Units, self).__init__()
```

Comment:
Agreed. This change is more correct so leave it. I just hope it doesn't cause an unexpected change.